internal/gengapic/testdata/foo_example.want

```diff
@@ -66,6 +70,9 @@ func ExampleFooClient_GetBigThing() {
 }
 
 func ExampleFooClient_GetManyThings() {
+	// import "mypackage" mypackagepb
```

Comment:
other way around - quotes around the package import, the name its imported as is unquoted

Reply:
That is how this is implemented, the example is just a mediocre expression of it. `mypackagepb` is in fact the name while `"mypackage"` is the import path. :sweat_smile:

Reply:
in that case, they need to be swapped!

Reply:
(sorry, i see that i messed that up in my suggestion above. whoops!)

Reply:
Done